### DISCUSSION TITLE
Un-Read Bookmarks

### DISCUSSION THREAD

**Discussion by DamQuangKhoa** [dailydotdev/daily#741] [Feature Requests]
There are only Bookmark tags...
I usually use Bookmark for an important articles.
But sometimes I found the articles I want to read but I don't have time so I want to add them to Un-Read Bookmarks.
I would very happy if we have that feature

**Comment by idoshamun** [comment]
Hi @DamQuangKhoa,

Thanks for the suggestion. Let's keep the discussion here: https://github.com/dailydotdev/daily/discussions/364
